### ISSUE TITLE
Alternative Milestone Display Option

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bardsoftware/ganttproject#188]
```
I think it would be a wonderful feature to have the option of displaying a 
milestone as a verticle line that cross the entire project.
```

Original issue reported on code.google.com by `hli...@gmail.com` on 9 Feb 2011 at 3:09

**Comment by AquaJew** [comment]
I think the issue is confusing what a "milestone" means. In the Gantt world, a Milestone is a marker when a feature can be reviewed. Usually it indicates when a project has reached a point of development, or when multiple features are integrated together. It's more of a landmark, perhaps?

What this feature request is asking for is the ability to **mark an important date on the calendar**. It definitely can be the date of a milestone, but there are a number of other uses; when other projects not being tracked are being released, press announcement days, etc.

What I want is similar to the ability to Add Holiday, but I want to add a day where work continues even though the color is different. I'm going to play around with the holiday settings and see if I can make it work.

**Comment by dbarashev** [comment]
You can add a color and label to any calendar date, no matter if it is working day or holiday. You can explicitly set some date as working, holiday or "neutral", add description and define color. 

See also http://ganttproject.blogspot.ru/2014/11/ganttproject-27-beta-iv.html

**Comment by AquaJew** [comment]
> You can add a color and label to any calendar date, no matter if it is working day or holiday. You can explicitly set some date as working, holiday or "neutral", add description and define color. ~ @dbarashev 

Yep, that's what I did, it's just (IMHO) in the wrong place as it's not really a holiday. It's just a specific date callout.

**Comment by dbarashev** [comment]
It sit sin the project calendar settings. I can't think of any better place. "Add holiday" is just a shortcut on the chart which creates one of possible events, namely, holiday. There is "Project Calendar..." action in the same context menu which opens the calendar editor.
